packages/connect-iframe/webpack/prod.webpack.config.ts

```diff
@@ -44,6 +44,24 @@ const config: webpack.Configuration = {
                     filename: './workers/sessions-background-sharedworker.[contenthash].js',
                 },
             },
+            {
+                test: /sharedLoggerWorker.ts/i,
```

Comment:
This is done by design. Because we want to load  the file from different environment with separated builds we would not know the hash to load the file. I am thinking about some solutions to this.